style.css

```diff
@@ -0,0 +1,184 @@
+  height: 100vh;
+  width: 100%;
+  min-height: 500px;
+  background-image: url("../images/background.png");
+  background-size: cover;
+  background-position: center;
+  background-color: rgb(255, 255, 255);
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 5px;
+  padding-left: 8%;
+}
+
+.logo {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #28352f;
+}
+
+.nav-list {
+  list-style: none;
+  padding: 10px 10px;
+  display: inline-flex;
+}
+
+nav ul li a {
+  text-decoration: none;
+  text-transform: capitalize;
+  font-weight: bold;
+  color: black;
+  font-size: 15px;
+}
+
+nav ul li a:hover {
+  color: green;
+}
+
+.btn {
+  background-color: green;
+  font-weight: bold;
+  color: black;
+  text-decoration: none;
+  border: 2px, solid transparent;
+  padding: 10px 25px;
+  border-radius: 30px;
+  transition: transform 0.4s;
+}
+
+.btn:hover {
+  transform: scale(1.2);
+}
+
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 38px;
+  line-height: 50px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #172b4d;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 38px;
+  line-height: 50px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: rgb(6, 202, 137);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.supporting-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 30px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #172b4d;
+  max-width: 500px;
+  margin: auto;
+  padding: 0.5rem;
+}
+
+.socialmedia {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  height: 20px;
+  margin: auto;
+  width: 100%;
+  padding: 2rem;
+}
+
+.disabled {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: calc(50% - 40px / 2 + 0.5px);
+  bottom: 4px;
+  background: #ffff;
+}
+
+.socialmedia:hover {
+  color: green;
+}
+
+.hambuger-btn {
+  display: block;
+  font-size: 25px;
+  font-weight: bold;
+  color: black;
+  float: right;
+  line-height: 70px;
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+#check {
+  display: none;
+}
+
+#check:checked ~ .nav-list {
+  height: 100vh;
+}
+
+#check:checked ~ .nav-list li {
+  opacity: 1;
+}
+
+@media (max-width: 700px) {
+  .nav {
+    height: 100vh;
+    background-color: gray;
+  }
+
+  .hambuger-btn {
+    position: relative;
+    height: 100px;
+    top: auto;
+    background-color: none;
+    float: right;
+  }
+
+  .nav-list {
+    display: none;
+  }
+}
+
+@media (max-width: 480px) {
+  .disabled {
+    position: relative;
+  }
+}
```

Comment:
- [ Optional  ] As the suggested by the previous reviewer  and stated in the project requirement  You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/193003740-95782926-84f9-4710-a5c6-717e22624ff7.PNG)

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/193003975-235d6a8a-2498-4401-8a88-908c802d38ab.PNG)

- [ ] Kindly make sure you use the correct styling on the above text like on the Figma design e.g.: color, fonts(size && weight), line height, paddings && margins,...